.devcontainer/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM --platform=linux/amd64 mcr.microsoft.com/devcontainers/base:jammy
+
+SHELL ["/bin/bash", "--login", "-i", "-c"]
+USER root
+
+RUN apt -y update
+RUN apt install -y make wget
+
+# Initialize nvm
+RUN wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash
+RUN source /root/.bashrc && nvm install --lts
+
+# Initialize go
+RUN wget -O go.tar.gz https://go.dev/dl/go1.23.1.linux-amd64.tar.gz
+RUN rm -rf /usr/local/go && tar -C $HOME -xzf go.tar.gz
+RUN rm -rf go.tar.gz
+RUN echo "export PATH=$PATH:$HOME/go/bin" >> $HOME/.bashrc
+
+# Install python
+RUN apt install -y \
+    build-essential \
+    zlib1g-dev \
+    libssl-dev \
+    libbz2-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+    curl \
+    libncursesw5-dev \
+    xz-utils \
+    tk-dev \
+    libxml2-dev \
+    libxmlsec1-dev \
+    libffi-dev \
+    liblzma-dev
+RUN git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+RUN echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bashrc
+RUN echo 'command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bashrc
+RUN echo 'eval "$(pyenv init -)"' >> ~/.bashrc
+RUN exec $SHELL
```

Comment:
I would be a little weary of this when run in a docker container. I feel like this could impose some issues.
Does it work if you source the file instead?

Reply:
Good point. Solved in 1dad3d9